qcodes/dataset/data_export.py

```diff
@@ -83,17 +85,17 @@ def get_data_by_id(run_id: int) -> List:
         dep_data_dict_index = None
 
         for param_name, data in data_dict.items():
-            data_dict = {}
+            my_data_dict: Dict[str, Union[str, np.ndarray]] = {}
 
-            data_dict['name'] = param_name
+            my_data_dict['name'] = param_name
```

Comment:
was the renaming really needed? :)

Reply:
Yes, have a look at the code please. The code was redefining data_dict while iterating over data_dict.

Reply:
yes, didn't notice it, sorry. that was terrible))